io.github.mmarco94.tambourine.yml

```diff
@@ -0,0 +1,60 @@
+app-id: io.github.mmarco94.tambourine
+runtime: org.freedesktop.Platform
+runtime-version: '22.08'
+add-extensions:
+  org.freedesktop.Platform.ffmpeg-full:
+    directory: lib/ffmpeg
+    version: '22.08'
+    add-ld-path: .
+sdk: org.freedesktop.Sdk
+sdk-extensions:
+  - org.freedesktop.Sdk.Extension.openjdk17
+command: /app/bin/run.sh
+modules:
+  - name: openjdk
+    buildsystem: simple
+    build-commands:
+      - /usr/lib/sdk/openjdk17/install.sh
+  - name: tambourine
+    buildsystem: simple
+    build-commands:
+      # ffsampledsp requires libbz2.so.1.0, however Flatpak only provides libbz2.so.1.0.8
+      - ln -s /usr/lib/x86_64-linux-gnu/libbz2.so.1.0.8 /app/jre/lib/libbz2.so.1.0
+      - # Installing the actual app
+      - install -D tambourine-linux-x64-1.0.0.jar /app/bin/tambourine.jar
+      - install -D run.sh /app/bin/run.sh
+    sources:
+      - type: file
+        url: https://github.com/MMarco94/tambourine-music-player/releases/download/v0.1/tambourine-linux-x64-1.0.0.jar
+        sha256: ffd356a016237992d4fe62fce3d401c5d625d24a14d106e642fd66409e76330e
+      - type: file
+        path: run.sh
+  - name: extras
+    buildsystem: simple
+    build-commands:
+      - install -D io.github.mmarco94.tambourine.desktop /app/share/applications/io.github.mmarco94.tambourine.desktop
+      - install -D io.github.mmarco94.tambourine.metainfo.xml /app/share/metainfo/io.github.mmarco94.tambourine.metainfo.xml
+      - install -D icon.png /app/share/icons/hicolor/256x256/apps/io.github.mmarco94.tambourine.png
+    sources:
+      - type: file
+        path: io.github.mmarco94.tambourine.desktop
+      - type: file
+        path: io.github.mmarco94.tambourine.metainfo.xml
+      - type: file
+        path: icon.png
+finish-args:
+  - --filesystem=xdg-music:ro
+  # TODO: remove once the file portal integration works
+  - --filesystem=host:ro
+  - --socket=wayland
+  - --socket=fallback-x11
+  - --socket=pulseaudio
+  - --share=ipc
+  - --device=dri
+  - --env=PATH=/app/jre/bin:/app/bin:/usr/bin
+  - --env=JAVA_HOME=/app/jre
+  # TODO: this shouldn't be used, but if I don't do this, the DBUS connection gets closed
+  # almost immediately because of "Invalid client serial"
+  - --socket=session-bus
```

Comment:
you should determine which name you need. This is NOT allowed.

Reply:
I know which one I need, the problem is that the Flatpak runtime closes the DBus connection with an `Invalid client serial` error after a while.

The only solution I found was to do this.

Reply:
I've also opened https://github.com/hypfvieh/dbus-java/issues/220 to get my issue with the dbus fixed